### DISCUSSION TITLE
VOX Delay

### DISCUSSION THREAD

**Discussion by brookbonde** [egzumer/uv-k5-firmware-custom#177] [Ideas]
Is there a possibility to add a VOX DELAY option?   I am looking to use this radio with VOX and an adjustment to extend the delay would be really useful.

**Comment by ErikS-web** [comment]
I miss the motivation and PURPOSE of the delay and also the step size.

But I have the impression that this is already an issue #90 

So please explain or confirm and this item can be closed.

**Reply by brookbonde** [reply]
Thank you, this item can be closed.
